src/tests/ftest/pool/create.py

```diff
@@ -112,3 +112,35 @@ def test_create_no_space_loop(self):
                 "Destroying a large capacity pool that spans across all but "
                 "the first server should succeed."
             )
+
+    def test_create_and_query(self):
+        """JIRA ID: DAOS-10339
+
+        Test Description:
+            TODO
+
+        :avocado: tags=hw,small
+        :avocado: tags=pool
+        :avocado: tags=pool_create_tests,create_and_query
+        """
+        # Create pool
+        self.add_pool(namespace="/run/pool_3/*")
+        epsilon_bytes = 1 << 20 # 1MiB
+        self.assertLessEqual(abs(self.pool.scm_per_rank - self.pool.scm_size.value), epsilon_bytes,
+                "SCM size of the pool created should not be too different from the given size")
+        self.assertGreaterEqual(self.pool.scm_per_rank, self.pool.scm_size.value,
+                "SCM size of the pool created should be at least greater than the given size")
+        self.assertEqual(self.pool.nvme_per_rank, self.pool.nvme_size.value,
+                "NVME size of the pool created should be equal to the given size")
+
+        # Query pool created
+        self.pool.set_query_data()
+        # FIXME Replace with nb of enabled ranks when the PR will be merged
```

Comment:
(pylint-fixme) FIXME Replace with nb of enabled ranks when the PR will be merged